api_app/tests_ma/test_api/test_routes/test_workspace_templates.py

```diff
@@ -40,7 +39,8 @@ class TestWorkspaceTemplate:
 
     @pytest.fixture(autouse=True, scope='class')
     def _prepare(self, app, admin_user):
-        app.dependency_overrides[get_current_tre_user] = admin_user
+        app.dependency_overrides[get_current_admin_user] = admin_user
+        app.dependency_overrides[get_current_workspace_owner_or_researcher_user_or_tre_admin] = admin_user
```

Comment:
is it correct that both L42 and L43 are 'adnin_user' ?

Reply:
That second override was redundant, removed.